MediaRecorder.bs

```diff
@@ -410,6 +405,67 @@ interface MediaRecorder : EventTarget {
   </table>
   </dd>
 
+  <dt><dfn method for="MediaRecorder"><code>replaceTrack(MediaStreamTrack track, MediaStreamTrack withTrack)</code></dfn></dt>
+  <dd>
+  When a {{MediaRecorder}} object&#8217;s {{replaceTrack()}} method is invoked,
+  the UA MUST run the following steps:
+  <ol>
+    <li>Let |recorder| be the {{MediaRecorder}} object on which the method was
+    invoked.</li>
+    <li>Let |track| be the method's first argument.</li>
+    <li>Let |withTrack| be the method's second argument.</li>
+    <li>If |recorder|'s {{state}} is {{inactive}}, return a promise rejected
+    with a newly created {{InvalidStateError}} {{DOMException}} and abort these
+    steps.</li>
+    <li>If |track| is not found in |recorder|'s [=[[Tracks]]=], return a promise
+    rejected with a newly created {{NotFoundError}} {{DOMException}} and abort
+    these steps.</li>
+    <li>If |track| is {{ended}}, or |withTrack|'s {{kind}} differs from
+    |track|'s {{kind}}, then return a promise rejected with a newly created
+    {{InvalidModificationError}} {{DOMException}} and abort these steps.</li>
+    <li>If the <a>isolation properties</a> of |track| or |withTrack| disallow
+    access from |recorder|, return a promise rejected with a newly created
+    {{SecurityError}} {{DOMException}} and abort these steps.</li>
+    <li>If the User Agent cannot record |withTrack| in place of |track| using
+    the current configuration, then return a promise rejected with a newly
+    created {{NotSupportedError}} {{DOMException}} and abort these steps.</li>
+    <li>Let |session| be the ongoing recording session corresponding to the
+    first occurance of |track| in |recorder|'s [=[[Tracks]]=].</li>
+    <li>Replace the first occurance of |track| in |recorder|'s [=[[Tracks]]=]
+    with |withTrack|.</li>
+    <li>Add |session| and |withTrack| as a pair to the list in |recorder|'s
+    [=[[ReplaceAtOnceList]]=] slot.</li>
+    <li>If |recorder|'s [=[[ReplacePromise]]=] slot is not <code>null</code>,
+    return it, and abort these steps.</li>
+    <li>Let |p| be a new promise.</li>
+    <li>Set |recorder|'s [=[[ReplacePromise]]=] slot to |p|, and queue
+    a task, using the DOM manipulation task source, that runs the following
+    steps:</li>
+    <ol>
+      <li>If |recorder|'s {{state}} attribute is {{inactive}}, resolve |p|, and
+      abort these steps.</li>
+      <li>Let |replaceList| be |recorder|'s [=[[ReplaceAtOnceList]]=], and set
+      |recorder|'s [=[[ReplaceAtOnceList]]=] to an empty list.</li>
+      <li>Set |recorder|'s [=[[ReplacePromise]]=] slot to <code>null</code>, and
+      run the remaining steps in parallel:</li>
+      <li>For each recording |session| and |withTrack| pair in |replaceList|,
+      instantly switch |session| to recording |withTrack| in place of what
```

Comment:
s/instantly/atomically/ ?

Reply:
Why is the term "seamlessly" https://w3c.github.io/webrtc-pc/#dom-rtcrtpsender-replacetrack not used here?

> switch seamlessly to transmitting withTrack instead of the sender's existing track

Reply:
It's in the note below. I based this on Henrik's https://github.com/w3c/mediacapture-record/pull/186. Happy to change it to be here if we prefer.

Reply:
Language in the body of a document is, in general, the primary source and controlling, not a note. 

https://github.com/w3c/webrtc-pc/issues/2171#issuecomment-484544660
> I don't think this note counts as citeable.

Since the opportunity is at hand would substitute "seamlessly" for "instantly" to avoid ambiguity in the text. Granted both "instantly" and "seamlessly" can also be parsed as there is no corresponding time frame referenced. If necessary substitute "SHOULD be an instant operation" for "SHOULD be a seamless operation".

Reply:
Unless "instantly" is indeed intended. Apparently that is the case given no `Promise` operation. "instantly" is the primary term used the term, as-applied in the specification, should be clearly defined.